corehq/apps/data_interfaces/tasks.py

```diff
@@ -213,9 +213,10 @@ def task_generate_ids_and_operate_on_payloads(
     repeater_id: Optional[str],
     domain: str,
     action,  # type: Literal['resend', 'cancel', 'requeue']  # 3.8+
+    state=None,
```

Comment:
Can the line above be uncommented out now? And could this line use something like `state: Literal[None, State.Pending, State.Cancelled]`?

Reply:
Yes, I think that makes sense for consistency with the surrounding code. https://github.com/dimagi/commcare-hq/pull/35047/commits/62a7cccac8f23e1330e258f09cbe28399ed66e5d